libgraphics/include/pivot/graphics/VulkanApplication.hxx

```diff
@@ -71,6 +63,11 @@ public:
     float getAspectRatio() const noexcept { return swapchain.getAspectRatio(); }
 
 private:
+    bool dispatchCulling(const CameraData &cameraData, const vk::CommandBufferInheritanceInfo &info,
+                         vk::CommandBuffer &cmd);
+    bool drawImGui(const CameraData &cameraData, const vk::CommandBufferInheritanceInfo &info, vk::CommandBuffer &cmd);
+    bool drawScene(const CameraData &cameraData, const vk::CommandBufferInheritanceInfo &info, vk::CommandBuffer &cmd);
```

Comment:
```suggestion
    bool dispatchCulling(const CameraData &, const vk::CommandBufferInheritanceInfo &, vk::CommandBuffer &);
    bool drawImGui(const CameraData &, const vk::CommandBufferInheritanceInfo &, vk::CommandBuffer &);
    bool drawScene(const CameraData &, const vk::CommandBufferInheritanceInfo &, vk::CommandBuffer &);
```

Reply:
Nope, won't work with clang-format (line limited to 120 columns, so that is why there is a line break).

Reply:
That's exactly why I suggest to remove the redundant parameters' names 😄

Reply:
Ah ! I missed that.